tools/run_test_api_server.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
Is this script used anywhere?

Reply:
It is meant to be run by user/developer that wants to run examples with the server. I'll describe it in the README in the future.

It is also run by `make test` after recent changes